examples/data-sources/auth0_resource_server/data-source.tf

```diff
@@ -0,0 +1,9 @@
+# An Auth0 Resource Server loaded using its identifier.
```

Comment:
```suggestion
# An Auth0 Resource Server loaded using its unique identifier.
```
As described in https://auth0.com/docs/api/management/v2#!/Resource_Servers/get_resource_servers_by_id.